packages/core/src/jwt.ts

```diff
@@ -12,6 +12,14 @@ export type JwtData<
   encoded?: string;
 };
 
+/**
+ * Options for the JWT verifier
+ */
+export type VerifierOptions = {
+  currentDate?: number; // current time in seconds since epoch
+  skewSeconds?: number; // allowed skew for the current time in seconds. Positive value that will lower the iat and nbf checks, and increase the exp check.
```

Comment:
This way it will be visible as documentation in code editors.

```suggestion
/**
 * current time in seconds since epoch
 */
  currentDate?: number;

 /**
  * allowed skew for the current time in seconds. Positive value that will lower the iat and nbf checks, and increase the exp check.
  */
  skewSeconds?: number;
```